lib/msf/core/auxiliary/jtr.rb

```diff
@@ -40,6 +40,11 @@ def initialize(info = {})
       ], Msf::Auxiliary::JohnTheRipper
     )
 
+    register_advanced_options(
+      [
+        OptBool.new('DELETE_TEMP_FILES',    [false, 'Delete temporary wordlist and hash files', true])
```

Comment:
I believe convention for advanced options is CamelCase `DeleteTempFiles` for what seems to be no clearly defined nor adequately explained reason.

Reply:
upon further review, you are correct!